### PR TITLE
Minor wording fixes.

### PR DESCRIPTION
**Pull request by sergetron**
Simplifies wording in the "Customizing basic execution parameters" section.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -236,10 +236,10 @@ You can specify your own values that replace the default service account (`Servi
 
 The example below customizes the following:
 
-- the default service account from `default` to `tekton`.
-- the default timeout from 60 minutes to 20 minutes.
-- the default `app.kuberrnetes.io/managed-by` label is applied to all Pods created to execute `TaskRuns`.
-- the default Pod template to include a node selector to select the node where the Pod will be scheduled by default. 
+- Service account from `default` to `tekton`.
+- Timeout from 60 minutes to 20 minutes.
```
Comment (vdemeester) [inline docs/install.md]: We may want to keep `default` in there in some ways. As it changes the default `timeout` or `serviceAccountName` the `TaskRun` will use if not specified in the `TaskRun` spec.

Comment (spomorski) [reply docs/install.md]: Restored the... defaults.  =)

**Comment by dibyom** [comment]
@sergetron This looks good...would you mind squashing up the commits?

**Comment by sergetron** [comment]
I'd have to do a new PR... is it unmergable right now?

**Comment by sbwsg** [comment]
Hey just wanted to pop in here and say that it makes life much harder when we come to perform releases if commits aren't squashed together.  This pain is recent for me as I've been doing the beta release candidates.

Building release notes is very time-consuming work and it's made more difficult when multiple commits refer to the exact same fix / feature / documentation issue because the release author has to go through them and de-dupe.

I'm happy to walk you through the squashing process when you've got time, let me know!

**Comment by sergetron** [comment]
Understood :) I'll take care of it!

**Comment by sergetron** [comment]
Closing as this is not essential during RC time.

**Pull request closed without merging**